src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,10 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
```

Comment:
Good job on defining the interface properly. Remember to add simple header comment :-).